AK/ByteBuffer.h

```diff
@@ -182,6 +182,20 @@ class ByteBuffer {
         return try_ensure_capacity_slowpath(new_capacity);
     }
 
+    /// Return a span of bytes past the end of this ByteBuffer for writing.
```

Comment:
`///`? Isn't that unusual?

Reply:
It's kind of a preparation for API documentation in the future. The `Core::Stream` classes also have similar comments.

Reply:
👍